Upstream: under review, https://github.com/libfuse/libfuse/pull/379

From 526953fa9ee8757d30efc17f9aa2cbd5aa72a7aa Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Sat, 9 Mar 2019 17:40:44 -0300
Subject: [PATCH] Define ALLPERMS for musl libc systems.

Like Void Linux musl and Alpine Linux
---
 test/test_syscalls.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/test_syscalls.c b/test/test_syscalls.c
index e9a5189..db4be56 100644
--- a/test/test_syscalls.c
+++ b/test/test_syscalls.c
@@ -14,6 +14,10 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
+#ifndef ALLPERMS
+# define ALLPERMS (S_ISUID|S_ISGID|S_ISVTX|S_IRWXU|S_IRWXG|S_IRWXO)/* 07777 */
+#endif
+
 
 static char testfile[1024];
 static char testfile2[1024];
-- 
2.21.0

